packages/cli/src/rpc/explain/collectContext.ts

```diff
@@ -118,6 +118,11 @@ export class ContextCollector {
     searchResponse: SearchRpc.SearchResponse;
     context: ContextV2.ContextResponse;
   }> {
+    if (this.vectorTerms.length === 0 || this.vectorTerms.every((term) => term.trim() === '')) {
```

Comment:
Yup you can do the same thing for Help context and even handle it also on the Navie side if you wish.

Reply:
I'll post a separate PR with a change to Navie